app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadFragment.java

```diff
@@ -2,11 +2,13 @@
 
 
 import android.app.Activity;
-import android.app.AlertDialog;
+import android.support.v7.app.AlertDialog;
+import android.app.Dialog;
 import android.content.Context;
 import android.database.DataSetObserver;
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
+import android.graphics.Color;
```

Comment:
Where are the changes?

Reply:
-import android.app.AlertDialog;
+import android.support.v7.app.AlertDialog;

This will remove the strange boundary around the alert window

Reply:
Could you please cleanup your commit history if these were all the changes made.

Reply:
@mhutti1 Shouldn't the extra area covered above the blue line be removed?

Reply:
Yeah but its not really relevant to this review.

Reply:
@mhutti1 Agreed. But why to deal with it separately if it can be solved right now.

Reply:
Well they can make the changes if they want but i'm not going to fail a pr over an unrelated change with no impact on the build.